addons/mail/static/src/components/message/message.scss

```diff
@@ -175,11 +175,11 @@
     &:hover, &.o-clicked {
 
         .o_Message_command {
-            display: flex;
+            opacity: 1;
         }
 
         .o_Message_commands {
-            display: flex;
+            opacity: 1;
```

Comment:
don't change this one, it was flex for display reasons I think, only the one that has opacity 0 should have opacity 1

Reply:
All `display: flex` in style were alternatives to `opacity`, so they are safe for removal